Sofa/framework/Type/src/sofa/type/Quat.h

```diff
@@ -36,48 +71,72 @@ constexpr qNoInit QNOINIT;
 template<class Real>
 class Quat
 {
-    Real _q[4];
+    std::array<Real, 4> _q{};
```

Comment:
```suggestion
    sofa::type::fixed_array<Real, 4> _q{};
```
But to verify if it does not introduce a side effect